tunnel.go

```diff
@@ -38,16 +33,17 @@ type Tunnel interface {
 	// Will be empty for non-labeled tunnels.
 	Labels() map[string]string
 
-	// Convert this tunnel to a [net.Listener].
-	AsListener() ListenerTunnel
 	// Use this tunnel to serve HTTP requests.
 	AsHTTP() HTTPTunnel
-}
 
-// A tunnel that also implements [net.Listener].
-type ListenerTunnel interface {
-	Tunnel
-	net.Listener
+	// Convenience method that calls `CloseWithContext` with a default timeout
+	// of 5 seconds.
+	Close() error
```

Comment:
Can we get rid of this since it's included in `net.Listener`, or is it here just for the doc comment?

Reply:
yeah just here for the doc comment which seems important enough to leave around. could be we even want to include Accept() for the comment as well